src/packages/tests/src/__tests__/integration/mssql/__database.ts

```diff
@@ -8,23 +8,24 @@ export const database = {
   datasource: {
     url: ctx => getConnectionInfo(ctx).connectionString,
   },
+  previewFeatures: ["microsoftSqlServer"],
   connect: ctx => {
     const credentials = getConnectionInfo(ctx).credentials
-    const pool = new sql.ConnectionPool({
-      user: credentials.user,
-      password: credentials.password,
-      server: credentials.server,
-      database: ctx.step === 'scenario' ? `master_${ctx.id}` : `master`,
-      pool: {
-        max: 1,
-      },
-      options: {
-        enableArithAbort: false,
-      },
-    })
+    const pool = new sql.ConnectionPool(credentials) // always connect to master to create the db
     return pool.connect()
   },
-  send: (pool, sql) => pool.request().query(sql),
+  create: async (pool, sqlUp) => {
+    await pool.request().query(sqlUp) // create database from master
+    pool.close()
```

Comment:
I don't think this `pool.close` should be needed, the harness will call `database.close`, which should close the pool anyway.

Reply:
The `newPool` you create below is being disposed off correctly, so..

Reply:
I think I need to explicitly close the pool created in the connect() inside send() otherwise the mssql tests hang when I run them locally. Seems like the state.db.close() called from the test harness only runs once after all the scenarios complete so we end up with too many open pools during execution.

Reply:
Oh okay!